knative-operator/pkg/common/util.go

```diff
@@ -42,6 +42,11 @@ func buildImageOverrideMapFromEnviron() map[string]string {
 		pair := strings.SplitN(e, "=", 2)
 		if strings.HasPrefix(pair[0], "IMAGE_") {
 			name := strings.SplitN(pair[0], "_", 2)[1]
+			// convert
+			// "IMAGE_eventing-controller__eventing-controller=docker.io/foo2"
+			// to
+			// eventing-controller/eventing-controller: docker.io/foo2
+			name = strings.Replace(name, "__", "/", 1)
```

Comment:
I'm wondering whether you need `__` at all. If we split on a single underscore, then I'm either going to have `IMAGE_<container>` or `IMAGE_<deployment>_<container>` right? So why do I need the double underscore? I know what I have based on how many fields I get after the split. Or am I missing something?

Reply:
@jcrossley3 no you're not missing anything. I wanted to have an obvious separator there.
Let me change it to a single `_`.